src/main/java/com/maxmind/minfraud/request/Email.java

```diff
@@ -74,11 +95,32 @@ public Email build() {
     }
 
     /**
-     * @return The MD5 of the email address used in the transaction.
+     * @return The email address field to use in the transaction. This will be
+     * a valid email address if you used {@link #address(String)}, an MD5 hash
+     * if you used {@link #hashAddress()} as well, or null if you did not set an
+     * email address.
      */
     @JsonProperty("address")
+    public String getAddress() {
+        if (address == null) {
+            return null;
+        }
+        if (hashAddress) {
+            return DigestUtils.md5Hex(address);
```

Comment:
I realize this isn't a new issue, but we should be lowercasing address before hashing it.